libdrgn/python/util.c

```diff
@@ -103,6 +103,26 @@ int index_converter(PyObject *o, void *p)
 	}
 }
 
+int string_int_converter(PyObject *o, void *p)
```

Comment:
Until we need this in more places, I'd rather fold it into `Program_symbol()`.

Reply:
I think it would be a little more messy to have this directly inline in `Program_symbol` (and it would make `Program_symbol` rather large/complex. I decided to move the function into program.c but keep it in it's own function (rather than inline) to make it easier to refactor out if needed. Does that sound fine?

Reply:
Hm it's only a couple more lines of code, right?

```c
if (PyUnicode_Check(arg)) {
	const char *name;

	name = PyUnicode_AsUTF8(arg);
	if (!name)
		return NULL;
	err = drgn_program_find_symbol_by_name(&self->prog, name, &sym);
} else {
	struct index_arg address = {};

	if (!index_converter(arg, &address))
		return NULL;
	err = drgn_program_find_symbol_by_address(&self->prog,
						  address.uvalue, &sym);
}
```

`Program_symbol()` is not doing much other than interpreting the argument and passing it through to `libdrgn`.